dotcom-rendering/playwright.config.ts

```diff
@@ -32,4 +38,12 @@ export default defineConfig({
 			use: { ...devices['Desktop Chrome'] },
 		},
 	],
+	webServer: {
```

Comment:
Playwright can run your server for you can check when it is available:

https://playwright.dev/docs/test-webserver